source/scripts/data/monsters.js

```diff
@@ -6,11 +6,27 @@ export default {
         color: DATA.COLORS.RED,
         health: 1,
         strength: 1,
+        movement: "simplechase"
     },
     BLUE_SLIME: {
         sprite: DATA.SPRITES.MONSTERS.SLIME,
         color: DATA.COLORS.BLUE,
         health: 2,
         strength: 1,
+        movement: "simplechase"
+    },
+    GREEN_BAT: {
+        sprite: DATA.SPRITES.MONSTERS.BAT,
+        color: DATA.COLORS.GREEN,
+        health: 2,
+        strength: 1,
+        movement: "wander-orthogonal"
+    },
+    PINK_BAT: {
+        sprite: DATA.SPRITES.MONSTERS.BAT,
+        color: DATA.COLORS.PINK,
+        health: 1,
+        strength: 2,
+        movement: "wander-diagonal"
```

Comment:
So instead of passing simple text enums, what would you think about making this a function?